pkg/entrypoint/entrypointer.go

```diff
@@ -232,38 +239,80 @@ func (e Entrypointer) Go() error {
 
 	// strings.Split(..) with an empty string returns an array that contains one element, an empty string.
 	// This creates an error when trying to open the result folder as a file.
-	if len(e.Results) >= 1 && e.Results[0] != "" {
+	if (len(e.Results) >= 1 && e.Results[0] != "") || (len(e.StepResults) >= 1 && e.StepResults[0] != "") {
```

Comment:
If this is an or, do we even need this check? If you can have Results without StepResults, and StepResults without Results, it should be okay to have neither StepResults or Results?